.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+## What is this PR doing?
+
+
+## How can these changes be manually tested?
```

Comment:
Can remove this point. A CI testing workflow (once it's implemented) should suffice.

Reply:
I'm not so sure about that. Perhaps the wording should change when there is CI but I'd suggest manual testing continues to be a thing. Especially for UI changes in the extension.

Also, do you see any reason for change before CI is actually added?

Reply:
yeah sounds good. let's keep it for now.